src/server/plugins/register-form/steps/registered-address.js

```diff
@@ -0,0 +1,46 @@
+import Joi from 'joi';
+
+import { postHandlerFactory, getHandlerFactory, dependsOnBoolean } from './common';
+import previouslyRegisteredStep from './previously-registered';
+
+const schema = Joi.object().keys({
+  'registered-address-correct': Joi.boolean().required()
+    .meta({
+      componentType: 'multiple-choice',
+      children: [
+        { label: 'Yes', value: 'true' },
+        { label: 'No', value: 'false' },
+      ],
+      variant: 'radio',
+    })
+    .options({
+      language: {
+        any: { required: '!!Please tell us if you’re registered with with this address' },
```

Comment:
It's the [error message syntax](https://github.com/hapijs/joi/blob/v10.2.2/API.md#validatevalue-schema-options-callback) for Joi. It's to override the error message. In some cases we want something that reads better rather than a generic "[field name] is required".